### PR TITLE
docs: optimize the docs and fix small errors / typos

### PR DESCRIPTION
**Pull request by meiserloh**

### CODE REVIEW

**Review by robertauer — CHANGES_REQUESTED** [review]

File: docs/additional/internal_aspects_en.md [hunk 1]
```diff
@@ -247,8 +247,8 @@ The `docs` folder is structured according to this exemplary scheme:
 │ ├── users_endpoint_en.md  
 │ └── users_endpoint_en.md  
 └── /development  
-└── how_to_create_an_image.md  
-└── how_to_build_a_plugin.md
+│ ├── how_to_create_an_image.md  
+└ └── how_to_build_a_plugin.md
```
[the remark this answers is not in the document]
Comment (meiserloh) [reply docs/additional/internal_aspects_en.md]: done

File: docs/core/basics_en.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Therefore, a container image must be created, which is then instantiated into a
       - Go-templating
       - Health checks
 - If the Dogu is a web application, the line `ENV SERVICE_TAGS=webapp` is necessary.
-  This will cause the new dogu to appear in the warp menu.
+  This will cause the new dogu to appear in the warp menu (along with the `warp`-tag in the dogu.json).
```
[the remark this answers is not in the document]
Comment (meiserloh) [reply docs/core/basics_en.md]: :heavy_check_mark:

File: docs/additional/interesting_aspects_en.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ When building images, the following aspects should be considered:
 [multistage-build]: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds
 [container-tools]: https://github.com/cloudogu/base/blob/3466b5e95c25a6c5ac569069167e513c71815797/Dockerfile#L10
 [base-images]: https://github.com/cloudogu/sonar/blob/8e389605d1f2fa7720d725a1cca6692f4c6b77e3/Dockerfile#L1
-[healthcheck]: https://github.com/cloudogu/sonar/blob/8e389605d1f2fa7720d725a1cca6692f4c6b77e3/Dockerfile#L3
+[healthcheck]: https://github.com/cloudogu/sonar/blob/8e389605d1f2fa7720d725a1cca6692f4c6b77e3/Dockerfile#L41
```
[the remark this answers is not in the document]
Comment (meiserloh) [reply docs/additional/interesting_aspects_en.md:62]: :heavy_check_mark:

File: docs/core/compendium_en.md [hunk 4]
```diff
@@ -303,7 +303,7 @@ type Dogu struct {
 
 Name contains the dogu's full qualified name which consists of the dogu namespace and the dogu simple name, delimited by a single forward slash "/". This field is mandatory.
 
-The dogu namespace allows to regulate access to dogus in that namespace. There are three reserved dogu namespaces: The namespaces `official` and `k8s` are open to all users without any further costs. In contrast to that is the namespace `premium` is open to subscription users, only.
+The dogu namespace allows to regulate access to dogus in that namespace. There are three reserved dogu namespaces: The namespaces `official` and `k8s` are open to all users without any further costs. In contrast, the `premium` namespace is open to subscription users, only.
```
[the remark this answers is not in the document]
Comment (meiserloh) [reply docs/core/compendium_en.md]: :heavy_check_mark:

**Pull request merged**